src/index.html

```diff
@@ -9,6 +9,7 @@
     />
   </head>
   <body>
+    <div id="tree"></div>
```

Comment:
The link tag is referencing a SCSS file (`main.scss`). Browsers cannot directly interpret SCSS files. You need to compile the SCSS file into a CSS file and link to the resulting CSS file instead.